examples/metrics/docker/prometheus.yaml

```diff
@@ -0,0 +1,5 @@
+scrape_configs:
```

Comment:
I'd be nice to have some documentation about how to access the Prometheus dashboard.

Reply:
I was thinking it will make sense to have it in #423, including all instructions on how to start Collector in docker, I can add a README as part of this PR as well

Reply:
BTW the URL to access Prometheus is http://localhost:9090/graph